algorithm/cgal/separateSurfaceMesh.js

```diff
@@ -2,18 +2,21 @@ import { getCgal } from './getCgal.js';
 
 export const separateSurfaceMesh = (
   mesh,
-  keepVolumes = true,
-  keepCavitiesInVolumes = true,
-  keepCavitiesAsVolumes = false
+  keepShapes = true,
```

Comment:
That's interesting that keeping only the holes is an option

Reply:
Yeah, it means that you can extract the cavities in a shape as shapes.

e.g., given Box(10).cut(Arc(3)) you could get back the Arc(3)